content/blog/2020-08-05-cml-self-hosted-runners-on-demand-with-gpus.md

```diff
@@ -0,0 +1,198 @@
+---
+title: CML self-hosted runners on demand with GPUs
+date: 2020-08-05
+description: |
+  A tutorial on how to setup your own self-hosted runners with GPUs using CML cloud-runners in your CML workflow in Github or Gitlab.
```

Comment:
please fit it under 80

+ add desriptionLong, see in some other posts how it's done. descriptionLong support basic Markdown and appears in the post before the title image. It's a summary usually, or can serve as an intro

Reply:
I would say in the descriptionLong let's try to expand a bit on what problem and how will people learn to solve it in the blog post.